x-pack/plugin/esql/qa/testFixtures/src/main/java/org/elasticsearch/xpack/esql/CsvTestsDataLoader.java

```diff
@@ -396,12 +401,27 @@ private static void loadDataSetIntoEs(
         boolean supportsSourceFieldMapping,
         boolean inferenceEnabled,
         IndexCreator indexCreator
+    ) throws IOException {
+        if (PARALLEL_THREADS > 1) {
```

Comment:
I figure we can `git revert` if we want the old way back. But I *do* think it's nice to make a constant for the threads like you've done. Maybe it should be related to the number of CPUs or something. I'm not sure how many CPUs the CI machines have these days.